python/ql/src/semmle/python/regex.qll

```diff
@@ -12,10 +12,14 @@ private predicate re_module_function(string name, int flags) {
     name = "subn" and flags = 4
 }
 
+/**
+ * Holds if `s` is used as a regex with the `re` module, with the regex-mode `mode` (if known).
+ * If regex mode is not known, `mode` will be `"None"`.
+ */
 predicate used_as_regex(Expr s, string mode) {
     (s instanceof Bytes or s instanceof Unicode)
     and
-    exists(ModuleValue re | re.getName() = "re" |
+    exists(ModuleValue re | re = Module::named("re") |
```

Comment:
Unless my eyes deceive me, `re` is only used one other place in the body of this `exists`, so maybe just do
`Module::named("re").attr(name)` on line 26 and dispense with the `exists` entirely?

(I tried doing this as a suggestion, but I couldn't get it to include the rest of this predicate.)

Reply:
sure, that should work! :+1: